modules/protocol/resources/META-INF/smithy/smithy4s.meta.smithy

```diff
@@ -43,3 +43,11 @@ structure indexedSeq {}
         :not(:test([trait|smithy4s.meta#indexedSeq],
                    [trait|smithy.api#uniqueItems]))""")
 structure vector {}
+
+// the errorMessage trait marks a structure's field as one that will be used
+// for the generated exception's error message.
+@trait(
+    selector: ":is(structure) > member",
```

Comment:
should we expect that the structure has the `@error` trait?

Reply:
yeah that'd make sense

Reply:
done, I'm trying to force the member to be a string shape, can't do it